docs/connectors/postgresql.asciidoc

```diff
@@ -965,19 +965,31 @@ PostgreSQL supports `NaN` (not a number) special value to be stored in the `DECI
 [[postgis-types]]
 ==== PostGIS types
 
-The PostgreSQL connector also has limited support for some of the http://postgis.net[PostGIS data types]
+The PostgreSQL connector also has full support for all of the http://postgis.net[PostGIS data types]
 
 [cols="20%a,15%a,30%a,35%a",width=150,options="header,footer",role="table table-bordered table-striped"]
 |=======================
 |PostGIS Data Type
 |Literal type (schema type)
 |Semantic type (schema name)
 |Notes
+|planar types, e.g. `POLYGON`, `MULTIPOINT`
+|`STRUCT`
+|`io.debezium.data.geometry.Geometry`
+|Contains a structure with 2 fields +
 
-|`POINT`
+* `srid (INT32)` - Spatial Reference System Identifier defining what type of geometry object is stored in the structure
+* `wkb (BYTES)` - a binary representation of the geometry object encoded in thw Well-Known-Binary format.
+Please see http://www.opengeospatial.org/standards/sfa[Open Geospatial Consortium Simple Features Access specification] for the format details.
+
+|`GEOGRAPHY` (spherical)
```

Comment:
Isn't `GEOMETRY` missing?

Reply:
@gunnarmorling Hmm, it seems that push witout commit is not the way how git works ;-)